presto-hive-common/src/main/java/com/facebook/presto/hive/zorder/ZOrder.java

```diff
@@ -32,9 +32,10 @@
     public static final int MAX_INPUT_DIMENSIONS = 10;
 
     // Assume encodingBits is 1-based
-    private final List<Integer> encodingBits;
-    private final int totalBitLength;
-    private final int maxBitLength;
+    final List<Integer> encodingBits;
```

Comment:
Hmmm, why is `private` removed? 🤔 In general we don't encourage field sharing between classes because it kind of indicates abstraction not clean.